internal/service/cloudfront/sweep.go

```
@@ -160,25 +160,23 @@ func sweepCachePolicies(region string) error {
 }
 
 func sweepDistributions(region string) error {
-	var result *multierror.Error
-
 	// sweep:
 	// 1. Production Distributions
 	if err := sweepDistributionsByProductionStaging(region, false); err != nil {
-		result = multierror.Append(result, err)
+		log.Printf("[WARN] %s", err)
 	}
 
 	// 2. Continuous Deployment Policies
 	if err := sweepContinuousDeploymentPolicies(region); err != nil {
-		result = multierror.Append(result, err)
+		log.Printf("[WARN] %s", err)
 	}
 
 	// 3. Staging Distributions
 	if err := sweepDistributionsByProductionStaging(region, true); err != nil {
-		result = multierror.Append(result, err)
+		log.Printf("[WARN] %s", err)
 	}
 
-	return result.ErrorOrNil()
+	return nil
```

Errors should return errors, not fail silently